Client/Frontend/Widgets/PhotonActionSheet/AppMenu.swift

```diff
@@ -109,9 +110,11 @@ extension PhotonActionSheetProtocol {
         let noImageMode = PhotonActionSheetItem(title: imageModeTitle, iconString: iconString, isEnabled: noImageEnabled) { action,_ in
             NoImageModeHelper.toggle(isEnabled: action.isEnabled, profile: self.profile, tabManager: self.tabManager)
             if noImageEnabled {
-                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .blockImagesDisabled)
+                Analytics.shared.browser(.disable, label: .blockImages)
+//                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .blockImagesDisabled)
```

Comment:
Shall we remove this line?

Reply:
This is part of the Firefox and their Telemetry framework. Commenting out is usually the way we remove them.

Reply:
Commenting this out is just for clarity, we deactivated the framework, so even if it is not commented it should not perform any action. But having it commented makes it clearer that we are not tracking other things.